packages/block-library/src/cover/style.scss

```diff
@@ -34,13 +34,9 @@
 		background-size: auto;
 	}
 
-	&.has-background-dim {
-		background-color: $black;
-
-		&::before {
-			content: "";
-			background-color: inherit;
-		}
+	&.has-background-dim::before {
+		content: "";
+		background-color: inherit;
```

Comment:
Should this have become `$black` because the inherited background-color was removed?

Reply:
If that were to be `$black` themes would have to write some quite different styles to have their colors apply on Covers. The removal was motivated by pretty much that issue #25290, the way it made theme background style selectors need extra specificity. As an example, from twentytwenty:
`:root .has-primary-background-color { ... }`
Whereas the [docs on defining those](https://developer.wordpress.org/block-editor/developers/themes/theme-support/#block-color-palettes) use a simple class selector.
 Also, the `inherit` here is for when there's a defined background color via the block attributes.

Reply:
I have a "fix" in https://github.com/WordPress/gutenberg/pull/26564, that I'd love your thoughts on.